sql/server.go

```diff
@@ -146,9 +146,7 @@ func (s *Server) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 // exec executes the call. Any error encountered is returned; it is the
 // caller's responsibility to update the reply.
-func (s *Server) exec(call sqlwire.Call) error {
-	req := call.Args
-	resp := call.Reply
+func (s *Server) exec(req *driver.Request, resp *driver.Response) error {
```

Comment:
This is removal of the `sqlwire.Call` struct which was only being used in this one location. I suppose I can switch to the `xx(req)(reply, error)` pattern as well.